test_scripts/test_hover.py

```diff
@@ -0,0 +1,10 @@
+from ..flight.drone.drone_controller import DroneController
+from ..flight import constants as c
+
+controller = DroneController(c.Drones.LEONARDO_SIM)
+
+controller.add_takeoff_task(c.DEFAULT_ALTITUDE)
+controller.add_hover_task(c.DEFAULT_ALTITUDE, 10, c.Priorities.MEDIUM)
```

Comment:
Don't hardcode constants, choose an approriate name for what `10` represents as a constant variable.